fragments/k8s/cdk/bundle.yaml

```diff
@@ -33,7 +33,7 @@ services:
     options:
       channel: 1.10/stable
     expose: true
-    constraints: "cores=4 mem=4G root-disk=16G"
+    constraints: "cores=2 mem=4G root-disk=16G"
```

Comment:
Can you elaborate on dropping worker to `cores=2`? That's not needed to make this work on azure right?

We agreed on 4 cores as the default some time ago, here: https://github.com/juju-solutions/bundle-canonical-kubernetes/issues/338